### ISSUE TITLE
Update to 5.0

### ISSUE THREAD

**Issue by dxgldotorg** [thopiekar/Cura-packaging#81]
Cura 5.0 recently came out, with updated Qt dependencies and a new version of the plugin SDK.

In addition, please upload 5.0 compatible versions of your plugins to the Cura Marketplace, as all your plugins have been removed due to being incompatible with 5.0/Qt6.

**Comment by LightTemplar** [comment]
And I also ask about packages for Ubuntu 22.04.

**Comment by gamersi** [comment]
@thopiekar We need 5.0 support.

**Comment by thopiekar** [comment]
Got something for you to test :slightly_smiling_face: 
Basically follow these instructions and install `com.ultimaker.cura` instead.

**Comment by thopiekar** [comment]
PS: I'll do some changes soon, like shrinking the package. There are some development files in there still, but at least it runs for me (AMD and Intel).

**Comment by iggy** [comment]
Maybe a note in the repo/your website to make it easier to find out you've discontinued this? I think I looked at 10 separate pages before finally stumbling on this (closed) issue that says it's discontinued.

**Comment by LightTemplar** [comment]
@iggy, like this? 
![image](https://user-images.githubusercontent.com/453783/190514049-de0d263e-71de-4e60-aa1a-2cda848425c8.png)

**Comment by iggy** [comment]
Yeah. Something exactly like that. Except in one of the top 5 Google results for "cura ppa". Or even a page linked from one of the top 5 results.
